blocks/related-blogs/related-blogs.js

```diff
@@ -17,80 +17,76 @@ async function enrichProfiles(rArticles) {
-  const json = await response.json();
-  const qresult = json.data.filter((obj) => obj.tags.includes(tag)).slice(0, maxAutoItems);
+function buildCard(card) {
+  const {
+    path, title, image, tags, profiles: authorProfile,
+  } = card;
 
-  const enrichedArticles = await enrichProfiles(qresult);
+  const cardElement = document.createElement('article');
+  cardElement.classList.add('teaser');
 
-  enrichedArticles.forEach((value) => {
-    const blogListItem = document.createElement('li');
-    blogListItem.classList.add('related-list-item');
-    const ttr = document.createElement('div');
-    ttr.classList.add('teasertopicrelated');
-    blogListItem.append(ttr);
-    const article = document.createElement('article');
-    article.classList.add('teaser-base');
-
-    article.innerHTML = `<div class="teaser">
-
-    <a href="${value.path}" target="_self" class="teaser-link">
-        <div class="teaser-container">
-            <div>
-                <h2 class="teaser-description">
-                    ${value.title}
-                </h2>
-            </div>
-        </div>
+  if (authorProfile.Image === '') authorProfile.Image = defaultAuthorImage;
+
+  cardElement.innerHTML = `
+    <p class="tags">${JSON.parse(tags).join(', ')}</p>
+    <a href="${path}" target="_self" class="teaser-link">
+      <h2 class="teaser-description">
+        ${title}
+      </h2>
     </a>
-    <div class="authorprofile">
-            <div class="authorprofile-container">
-                <div class="authorprofile-image">
-                    <div class="nc-image-base">
-                        <div class="nc-image-container " itemscope="" itemtype="http://schema.org/ImageObject">
-                            <img class="nc-image" src="${value.profiles.Image ?? defaultAuthorImage}" itemprop="contentUrl" alt="" sizes="10vw">
-                        </div>
-                    </div>
-                </div>
-                <div class="authorprofile-info">
-                    <div class="authorprofile-name">${value.profiles.Name ?? defaultAuthorName}</div>
-                    <div class="authorprofile-position">${value.profiles.Title ?? defaultAuthorTitle}</div>
-                </div>
+    <div class="authorprofile-container">
+      <div class="authorprofile-image">
+        <div class="nc-image-base">
+            <div class="nc-image-container " itemscope="" itemtype="http://schema.org/ImageObject">
+                <img class="nc-image" src="${authorProfile.Image ?? defaultAuthorImage}" itemprop="contentUrl" alt="" sizes="10vw" />
             </div>
-    </div>
-</div>`;
+        </div>
+      </div>
+      <div class="authorprofile-info">
+          <div class="authorprofile-name">${authorProfile.Name ?? defaultAuthorName}</div>
+          <div class="authorprofile-position">${authorProfile.Title ?? defaultAuthorTitle}</div>
+      </div>
+    </div>`;
+
+  const pictureElement = createOptimizedPicture(image);
```

Comment:
pictures created with `createOptimizedPicture()` should either have an aspect-ratio set in the stylesheet or width and height attributes set to the img element. otherwise they will cause a layout shift. 

Also it would probably make sense to use a smaller width (pre default 750px) - something that better fits the design.

Reply:
fixed